react/react2/week1/search-app/git-app/src/components/Home.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useRef, useEffect } from 'react';
+import SearchUser from './SearchUser';
+import './main.css';
+
+function Main() {
+  const [username, setUsername] = useState('');
+  const handleInputChange = (e) => {
+    setUsername(e.target.value);
+  };
+  const inputRef = useRef('');
+  useEffect(() => {
+    inputRef.current.focus();
+  }, [inputRef]);
+
+  return (
+    <div>
+      <h1>Github User Serach</h1>
+      <input
+        type="text"
+        ref={inputRef}
+        onChange={handleInputChange}
+        placeholder="serach user"
+      ></input>
+      {username.length === 0 ? (
```

Comment:
this should be shown if there are no results based on your search query, not if the query is empty